client.go

```diff
@@ -232,20 +235,20 @@ func (c *Client) getTunnelOwnerUsername() string {
 }
 
 // listSharedTunnels returns tunnel states per user in the org with shared tunnels.
-func (c *Client) listSharedTunnels(protocol Protocol) (map[string][]TunnelState, error) {
+func (c *Client) listSharedTunnels(ct ClientType) (map[string][]TunnelState, error) {
 	states := make(map[string][]TunnelState)
-	url := fmt.Sprintf("%s/%s/tunnels?full=1&all=1&backend=%s", c.BaseURL, c.getTunnelOwnerUsername(), protocol)
+	url := fmt.Sprintf("%s/%s/tunnels?full=1&all=1&backend=%s", c.BaseURL, c.getTunnelOwnerUsername(), ct.GetBackends())
```

Comment:
I wish we had `protocol=` param here and not `backend=` 🤔

Reply:
I was thinking of switching this. We use `backend` extensively in t-m so a full switch will be tricky, but I'll start with the public facing API

Reply:
`protocol=` implemented